frontend/javascripts/types/api_flow_types.ts

```diff
@@ -610,8 +610,17 @@ export type ServerVolumeTracing = ServerTracingBase & {
   // https://github.com/scalableminds/webknossos/pull/4755
   resolutions?: Array<Point3>;
   organizationName?: string;
+  mappingName?: string | null | undefined;
+  mappingIsEditable?: boolean;
 };
 export type ServerTracing = ServerSkeletonTracing | ServerVolumeTracing;
+export type ServerEditableMapping = {
+  createdTimestamp: number;
+  version: number;
+  mappingName: string;
+  // The id of the volume tracing the editable mapping belongs to
+  tracingId: string;
+};
```

Comment:
what do you think about defining `EditableMapping` in store.ts using this definition? Sth like

```
type EditableMapping = ReadOnly<ServerEditableMapping> & {
    readonly type: "mapping";
}
```